composer.json

```diff
@@ -19,7 +19,7 @@
   "require-dev": {
     "ext-sqlite3": "*",
     "fakerphp/faker": "^1.18",
-    "phpunit/phpunit": "^6.0|^8.0|^9.0|^10.0|^11.0"
+    "phpunit/phpunit": "^6.0|^8.0|^9.0|^10.0|^11.0||^12.0"
```

Comment:
This should just be a single pipe character

Reply:
Thanks Chris.